src/Tizen.Applications.ComponentManager/Tizen.Applications/ComponentRunningContext.cs

```diff
@@ -0,0 +1,309 @@
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the component is terminated or not.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsTerminated
+        {
+            get
+            {
+                bool isTerminated = false;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentContextIsTerminated(_contextHandle, out isTerminated);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IsTerminated of " + _componentId + ". err = " + err);
+                }
+
+                return isTerminated;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the component is running as sub component of the group.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsSubComponent
+        {
+            get
+            {
+                bool isSubComponent = false;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentContextIsSubComponent(_contextHandle, out isSubComponent);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IsSubComponent of " + _componentId + ". err = " + err);
+                }
+
+                return isSubComponent;
+            }
+        }
+
+        /// <summary>
+        /// Resumes the running component.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of the system error.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of ouf of memory.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when failed because of permission denied.</exception>
+        /// <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+        /// <since_tizen> 6 </since_tizen>
+        public void Resume()
+        {
+            Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentManagerResumeComponent(_contextHandle);
+            if (err != Interop.ComponentManager.ErrorCode.None)
+            {
+                throw ComponentManager.ComponentManagerErrorFactory.GetException(err, "Failed to Send the resume request.");
+            }
+        }
+
+        /// <summary>
+        /// Pauses the running component.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of the system error.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of ouf of memory.</exception>
```

Comment:
Spelling, should be `out` instead of `ouf`.

Reply:
Done